### DISCUSSION TITLE
Θα υπάρξει μείωση του βαθμού από την εγκατάσταση non-systemd συστήματος σε Virtual Machine?

### DISCUSSION THREAD

**Discussion by KonstantinosTourtsakis** [courses-ionio/sw#1166] [Q&A] [answered]
Καλησπέρα, μετά από συζήτηση που είχα με συμφοιτητή σχετικά με την εγκατάσταση λετουργικού συστήματος χωρίς systemd δεν υπήρχε κοινή συμφωνία σχετικά με τις βαθμολογικές επιπτώσεις που μπορεί να έχει η εγκατάσταση αυτή σε εικονική μηχανή.

Στις [οδηγίες](https://courses-ionio.github.io/help/cli/) αναφέρει πως η εγκατάσταση μπορεί να γίνει είτε σε hardware είτε σε εικονική μηχανή και θα υπάρξει μείωση του βαθμού μόνο σε περίπτωση που χρησιμοποιήσουμε κάποιο φιλικότερο σύστημα όπως τα BSD.
> Η εγκατάσταση μπορεί να γίνει σε εικονική μηχανή ή σε κανονικό υλικό, και για μειωμένο βαθμό σε (free, open, etc)-BSD

Ισχύουν όσα αναφέρονται στις οδηγίες ή θα υπάρξει μείωση του βαθμού λόγω εγκατάστασης σε εικονική μηχανή?

**Comment by StavroulaAnatolaki** [comment]
Ένας φοιτητής είπε πως θα υπάρχει μια επιβάρυνση αν χρησιμοποιήσεις VM, δεν θυμάμαι ποιος.Όμως νομίζω πως αυτή είναι μια ερώτηση για τον καθηγητή μιας και στις οδηγίες δεν υπάρχει καθαρή εξήγηση.

**Reply by Angeloth1** [reply]
Το discussion το οποίο αναφέρει η συμφοιτήτρια #1162 στο οποίο είχα απαντήσει εσφαλμένα πράγμα όπου διόρθωσα για να μην μακρηγορώ η απάντηση που ψάχνεις είναι αυτή:
Ο καθηγητής στις οδηγίες αναφέρει το εξής
> Η εγκατάσταση μπορεί να γίνει σε εικονική μηχανή ή σε κανονικό υλικό...

μπορείς να διαβάσεις όλο το doc [εδώ](https://courses-ionio.github.io/help/cli/)

**Reply by epidrome** [reply]
@Angeloth1 γιατί δεν απαντάς κανονικά ώστε να πιστωθείς την λύση;

**Comment by epidrome** [comment, marked as answer]
για το μάθημα του λογισμικού είναι δεκτή η εικονική μηχανή
